### ISSUE TITLE
Recompiler build is only suitable for upgrades

### ISSUE THREAD

**Issue by zedtoo-de** [myelin/rpcemu-macos#13]
The recompiler build (I haven't checked the other one, sorry) is only suitable for upgrades as it does not include a "default data dir". New users would not know what is needed in the data directory.

**Comment by gingerbeardman** [comment]
Same goes for the interpreter build. So far the release is binary-only, not a full package.

Also related: #4

**Comment by zedtoo-de** [comment]
Point taken. I thought it was sufficiently separate as I was referring to new users rather than people upgrading an old installation.

**Comment by gingerbeardman** [comment]
@zedtoo-de you're absolutely correct and I think the this repo should/will produce a full package before it offers the download to the RPCEmu website as in #7
